jme3-xr/src/main/java/com/jme3/input/xr/Eye.java

```diff
@@ -0,0 +1,94 @@
+package com.jme3.input.xr;
+
+import com.jme3.app.SimpleApplication;
+import com.jme3.material.Material;
+import com.jme3.math.ColorRGBA;
+import com.jme3.math.Quaternion;
+import com.jme3.math.Vector3f;
+import com.jme3.renderer.Camera;
+import com.jme3.renderer.ViewPort;
+import com.jme3.scene.Geometry;
+import com.jme3.scene.shape.Box;
+import com.jme3.texture.FrameBuffer;
+import com.jme3.texture.Texture;
+import com.jme3.texture.Texture2D;
+import com.jme3.texture.FrameBuffer.FrameBufferTarget;
+import com.jme3.texture.Image.Format;
+
+public class Eye {
+	static int index = 0;
```

Comment:
Looks like there may be some tabs vs spaces issues going on here

Reply:
I can fix this, when I am back from holiday next week.
Have no access to my account currently.